Main.vb

```diff
@@ -1,6 +1,8 @@
 ﻿
 Module Test
     Public Sub Main()
+        ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls11
```

Comment:
https://msdn.microsoft.com/en-us/library/system.net.securityprotocoltype(v=vs.110).aspx says this property can be a bitwise mask. Can we do
```
ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls11 | SecurityProtocolType.Tls11
```